src/acl/RegexData.cc

```diff
@@ -42,11 +42,13 @@ ACLRegexData::lineOptions()
 }
 
 bool
-ACLRegexData::match(char const *word)
+ACLRegexData::match(char const *aWord)
 {
-    if (!word)
+    if (!aWord)
         return 0;
 
+    const SBuf word(aWord);
```

Comment:
It is not possible to simply extend regcomp()-based RegexPattern (and related classes) to support SBuf without secret and expensive to-SBuf conversions like this one. Please refactor to avoid these conversions after #1232 is merged.

Reply:
The problem you noticed has nothing to do with the internals of `RegexPattern` and will not be solved by anything in PR #1232.

It is the natural side-effect of config parser and ACL implementations still using `char *` instead of `SBuf`.

Reply:
> The problem you noticed has nothing to do with the internals of `RegexPattern` and will not be solved by anything in PR #1232.

I disagree on both counts.